group-generators/helpers/data-providers/snapshot/provider.ts

```diff
@@ -39,13 +47,12 @@ export default class SnapshotProvider
   }
 
   public async querySpaceVoters(
-    input: QuerySpaceVotersInput,
-    defaultValue = 1
```

Comment:
Thanks for pointing that out - I initially re-used this function but then decided to duplicate it and not use it. I forgot to change it back - I've changed it back.

Reply:
I was initially reusing the function in my queryspacevotersabovex, then I decided to recreate it a little bit differently. I forgot to change it back, I've done that now

Reply:
Maybe I didn't make myself clear sorry

First, no problem at all sir, it wasn't specified in the issue so it's not your fault ;))
It's just that I I very liked the fact that you use the "number of follows the user has" rather than a classic default value! So I instantly thought that it would be even better if the user can choose between this 2 options (default value or number of followers).

=> If the user doesn't set a defaultValue, by default the value of the account will be the number of vote.
So maybe the defaultValue is not the right name for this argument. Maybe "forcedValue" is better?

So it would be so nice to have this 2 choices for all the function that have a "special" account value (number of votes, number of proposal... etc).
In other words for these 4 functions:
```
querySpaceVoters
querySpaceAuthors
querySpaceVotersAboveX
queryProposalAuthorsAboveX
```

Reply:
Sure, a few questions

1. what use case would someone set the forced value?
2. do you want this in the factory interface? if so, can you specify the interface schema attributes please?

Reply:
1. Let's take an example:
I want to get all the voters of the aave space AND get all the onchain voters of AAVE.
But I want to give onchain voters an advantage over snapshot voters, so I'll set a forcedValue of 1 for snapshot voter, and a value of 2 for onchain voters.

2. Yes 👍 And sure, where will be for example the schema of the `querySpaceAuthors`
```
    {
      "name": "Get authors of space",
      "functionName": "querySpaceAuthors",
      "countFunctionName": "querySpaceAuthorsCount",
      "description": "Returns all authors of a specific space on Snapshot",
      "args": [
        {
          "name": "space",
          "argName": "space",
          "type": "string",
          "example": "sismo.eth",
          "description": "A specific space identifier"
        },
        {
          "name": "Forced value",
          "argName": "forcedValue",
          "type": "number",
          "example": "1",
          "description": "Force a specific value for all the accounts fetched"
        }
      ]
    },
```